labman/gui/static/js/plateViewer.js

```diff
@@ -451,7 +467,6 @@ PlateViewer.prototype.updateDuplicates = function () {
       var row = elem[0] - 1;
       var col = elem[1] - 1;
       that.wellClasses[row][col].push('well-duplicated');
-      that.data[row][col] = elem[2];
```

Comment:
As far as I can tell, this update is not ncessary since this check is should only update the "styling" of the cells, not the content.